packages/search/src/common/hub-type-map.ts

```diff
@@ -0,0 +1,117 @@
+interface IHubTypeMap {
```

Comment:
Let's reuse the existing code at https://github.com/Esri/hub.js/blob/master/packages/common/src/categories.ts or https://www.npmjs.com/package/@esri/arcgis-hub-content-types.